grammars/php.cson

```diff
@@ -2091,6 +2091,7 @@
       {
         'match': '''(?x)
           (?<!\\$)(?:
```

Comment:
What happens if one does not check for preceding/following dollar sign? Is there any kind of valid code that benefits from this check?

Reply:
You're correct that no valid code benefits.  It does, however, prevent invalid code like `$0` from being tokenized as somewhat valid.